src/puppetlabs/ezbake/core.clj

```diff
@@ -18,7 +19,9 @@
             [schema.core :as schema]
             [puppetlabs.ezbake.dependency-utils :as deputils]
             [puppetlabs.ezbake.exec :as exec]
-            [puppetlabs.config.typesafe :as ts]))
+            [puppetlabs.config.typesafe :as ts]
+            [cheshire.core :as json]
+            [puppetlabs.kitchensink.core :as ks]))
```

Comment:
Doesn't look like we ended up using the `ks` library so we should get rid of this.

Also, the dependencies here are grouped, so I'd move the `cheshire.core` line up out of the puppetlabs group down here and up towards the top with the other clojure/clj libraries.